New documentation content (#671)
graknlabs/docs: 0001-typedb/02-dev/05-read.md

if we want to prevent that we should return not only attributes, but also the entities that own the attributes in the 
`get` clause. For example:

```typeql 
match $f isa file, has size-kb $s; get $f, $s;
```

This will return pairs of `$f` and `$s`, including all duplicated values of `$s` (every `file` instance with its 
attribute of `size-kb`. Including all of the same values owned by different files. For example, all three values of 10 
will be returned this way).
</div>

### Count

Use the count keyword to get the number of the specified matched variable.

```typeql 
match
  $o isa object, has path $fp;
get $o, $fp; count;
```

<div class="note">
[Note]
The `count` function is applied to every result returned. If more than one variable mentioned in get, than count 
will show the number of unique results. This is also the case, when no `get` clause added, which actually means that 
all matched variables are included.
</div>

```typeql 
match
  $pe ($x, $y) isa permission;
  $x isa person, has full-name $x-n;
  $y (accessed-object: $o, valid-action: $act) isa access;
  $act has action-name $act-n;
  $o has path $o-fp;
get $x-n, $act-n, $o-fp; group $o-fp; count;
```

> This query returns the total count of person instances that has full-name, any access to an object with path and with 

Reviewer: ```suggestion
This query returns the total count of `person` instances that have (a) a `full-name` attribute and (b) any access to an object with a `path` with 
```
